sources/api/netdog/test_data/networkd/network/vlancfgdev.network

```diff
@@ -8,3 +8,4 @@ DHCP=ipv4
 [DHCPv4]
 UseDNS=true
 UseDomains=true
```

Comment:
nit: can we drop this from the default generated `.network` files and leave it up to the drop-ins to write? I'd rather not set the value in one place and override it in another.